packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -38,7 +47,8 @@ export interface SearchInputProps extends Omit<React.HTMLProps<HTMLDivElement>,
   onChange?: (value: string, event: React.FormEvent<HTMLInputElement>) => void;
   /** A suggestion for autocompleting */
   hint?: string;
-
+  /** Object that makes the search input expandable/collapsible */
+  expandableProps?: expandableProps;
```

Comment:
As discussed on slack.  I change this prop name to not be suffixed with "Props".